### DISCUSSION TITLE
'An unexpected internal error occurred' when deploying to vercel.

### DISCUSSION THREAD

**Discussion by AlviDervishaj** [vercel/community#1416] [Help] [answered]
### Summary

When I deploy my project on vercel, it runs into this error:
`

Traced Next.js server files in: 1.391s
17:08:41.586 | Created all serverless functions in: 2.797s
17:08:41.795 | Collected static files (public/, static/, .next/static): 13.664ms
17:08:42.914 | Build Completed in /vercel/output [1m]
17:08:50.353 | An unexpected internal error occurred

`

I have tried redeploying and pushing new changes so it automatically deploys when I push to GitHub but for the last 2 hours, it runs into this error.

When I searched this topic in discussions, it was resolved after a while. ( DATE : Mar 3, 2022 )

**Comment by swarnava** [comment, marked as answer]
Hi,

This should be fixed now: https://www.vercel-status.com/incidents/4gyl0x5r8wgh

**Reply by AlviDervishaj** [reply]
Yes, it is fixed now. 

Thank you for your time ⚡ 😄
